src/store/mock-state-service/mock-state.ts

```diff
@@ -1,14 +1,32 @@
 import { IBuilding } from '../../api/buildings/buildings-api';
+import { TechTreeItemType } from '../../api/tech-tree-item-api';
 import { ITech } from '../../api/techs/techs-api';
 import { IUnit } from '../../api/units/units-api';
-import { TechTreeItemType } from '../../api/tech-tree-item-api';
 import { FetchStatus } from '../fetch-status-service';
 import { FilterMode } from '../slices/tech-tree-filter-slice';
 import { SortBy } from '../slices/tech-tree-filter-slice/tech-tree-filter-service/sort-service';
 
 import { RootState } from '../';
 
 export const MOCK_STATE: RootState = {
+  auth: {
+    signupStatus: FetchStatus.FULFILLED,
+    loginStatus: FetchStatus.FULFILLED,
+    logoutStatus: FetchStatus.INIT,
+    refreshStatus: FetchStatus.FULFILLED,
+  },
+  users: {
+    user: {
+      id: 1,
```

Comment:
Question: do you want this in the initial state?

Reply:
for now yes, we can update later when we focus on testing further